src/clique/CliqueManager.cc

```diff
@@ -368,23 +368,18 @@ ncclResult_t CliqueManager::GetNumChannelsToUse(ncclFunc_t const coll,
     {
       // NOTE: These are currently based on collected data and not necessarily ideal for all hardware
       int numChannels;
-      switch (m_gcnArch)
-      {
-      case 906:
+      if (ArchName(m_gcnArchName, "gfx906")) {
```

Comment:
IsArchMatch?

Reply:
... how in the hell did that even compile?